openapi-generator/build.gradle.kts

```diff
@@ -30,13 +32,25 @@ dependencies {
 	testImplementation(Dependencies.Kotlin.Test.junit)
 }
 
-task("generateSources", JavaExec::class) {
+val openApiFile = file("../openapi.json")
+tasks.register("generateSources", JavaExec::class) {
 	main = application.mainClassName
 	classpath = sourceSets.main.get().runtimeClasspath
 
 	args = mapOf(
-		"openApiFile" to file("../openapi.json"),
+		"openApiFile" to openApiFile,
 		"apiOutputDir" to file("../jellyfin-api/src/main/kotlin-generated"),
 		"modelsOutputDir" to file("../jellyfin-model/src/main/kotlin-generated")
 	).map { listOf("--${it.key}", it.value.toString()) }.flatten()
 }
+
+arrayOf("stable", "unstable").forEach { flavor ->
+	tasks.register("downloadApiSpec${flavor.capitalize()}", de.undercouch.gradle.tasks.download.Download::class) {
```

Comment:
You could use an import for the `Download` class, would make the line a bit shorter.

Reply:
I'm not a fan of imports in script files, I think it looks ugly. But I'll add them.